vaadin-spring/src/main/resources/META-INF/additional-spring-configuration-metadata.json

```diff
@@ -0,0 +1,40 @@
+{
+  "properties": [
+    {
+      "name": "vaadin.productionMode",
```

Comment:
I'm not sure if I understand the suggestion.
The parameter should be according to the spring format: `vaadin.production-mode`

Currently it's `vaadin.productionMode` because the property name comes from vaadin flow. It's the same name in a Vaadin project without Spring prefixed by vaadin.

Do you mean that if I change the metadata to `"name":` `"vaadin.production-mode"` the existing application will still work with `vaadin.productionMode` ?

Reply:
The canonical format is in no way an "enforced" format but the format you must use so that your metadata is compilant.

> Do you mean that if I change the metadata to "name": "vaadin.production-mode" the existing application will still work with vaadin.productionMode ?

Yes.

Reply:
I've just created a ticket about it, thanks to your comment.
https://github.com/vaadin/spring/issues/637